### ISSUE TITLE
Saved flight times being reset and save file overwritten.

### ISSUE THREAD

**Issue by Ookami313** [rursache/InFlight-TWW#15]
Not sure if there is anything that can be done but wanted to report it just in case. With each update of the addon the .lua save file in the SavedVariables folder keeps getting reset and overwritten. When I log into the game the save file I have (made backup) will be roughly 57KB. When I log out, instead of updating the save file it gets overwritten and will reset to 2KB with the original 57KB file becoming a .lua.bak file. Logging back in, I have no saved flight path times, at least for Dragonflight flight points. Any help would be appreciated. Thanks.

![image](https://github.com/user-attachments/assets/5d13871d-f5e3-423a-a5ee-024e541f80ce)

**Comment by rursache** [comment]
@superwario15 if you have time maybe we can get to the bottom of this?

**Comment by Ookami313** [comment]
Willing to give it a shot. What do you need?
Edit: I will admit that I was off on the no DF flight times. While running around working on a quest I did notice that some DF flight times were showing like normal, I will assume because they were merged into the default.lua file.

[InFlight.txt](https://github.com/user-attachments/files/16639939/InFlight.txt)
This a copy of the file that was created when I logged off last night if it helps any.

**Comment by superwario15** [comment]
Could you share those other 2 InFlight.lua files as well?

**Comment by Ookami313** [comment]
[InFlight - Copy.txt](https://github.com/user-attachments/files/16642114/InFlight.-.Copy.txt)
The 57KB .bak version was overwritten after I logged out for the evening which left me with two 2KB reset versions. This is the same file. It's a copy from the backup I made of all my files from 10.2.6 roughly the day that 11.0.0 was released just in case there were any issues with new versions of the addon being released. 

Edit: Thinking about it, I think I was still using the original version InFlight at that time, if that info gives more to work with. I'm not sure if the original and this new version have any major differences that may play a part in this issue.

**Comment by superwario15** [comment]
Yeah, that was going to be my next question. The issue stems from the "dbinit" value in your InFlight.lua file. In the current version of the InFlight addon, the "dbinit" value is 1015. Since there's a mismatch, the addon would clear and start the InFlight.lua file from scratch. The old version of the addon was 920, and in that one you just shared it was also 920.

You _could_ just change that "dbinit" value to 1015 in your old file and the addon shouldn't rewrite what you have, but IMO the Defaults.lua is fairly comprehensive for DF flight times and it might just be better in the long-run to keep going with the new file and build that out and contribute new flight times normally.

**Comment by Ookami313** [comment]
OK, knowing that, that may be the way to go then. I did find a few that had not been mapped after the save was reset so I may just go back and fill those in just because I'm a stickler, but since DF basically only has about a week of life left I'll probably do as suggested and move on building a new set. I guess my main worry were some of what I had mapped being lost, especially outside of DF in other zones as well, however if the default.lua is being regularly updated my worries might be irrelevant any ways. Guess that solves the issue unless you have anything else. appreciate the help.

**Comment by superwario15** [comment]
Glad I could be of some assistance.

**Comment by superwario15** [comment]
@rursache This can be closed. This doesn't seem to be a bug, just a leftover symptom of version changes when the user changed over from a previous version.
